OpenRA.Game/FieldSaver.cs

```diff
@@ -73,31 +73,20 @@ public static string FormatValue(object v)
 
 			var t = v.GetType();
 
-			if (t == typeof(Color))
-			{
-				return ((Color)v).ToString();
-			}
+			if (v is Color c)
+				return c.ToString();
```

Comment:
Can we remove this completely and let the default `v.ToString()` at the end of the method handle this?

Color and Rectangle used to need custom handlers here because we were using the System.Drawing implementations that didn't output suitable strings. We could also change `Rectangle.ToString` and then do the same thing.